x-pack/plugins/case/server/routes/api/cases/find_cases.ts

```diff
@@ -97,9 +97,44 @@ export function initFindCasesApi({ caseService, router }: RouteDeps) {
           caseService.findCases(argsOpenCases),
           caseService.findCases(argsClosedCases),
         ]);
+
+        const totalCommentsFindByCases = await Promise.all(
+          cases.saved_objects.map(c =>
+            caseService.getAllCaseComments({
+              client,
+              caseId: c.id,
+              options: {
+                fields: [],
+                page: 1,
+                perPage: 1,
```

Comment:
Why one perPage?

Reply:
look below and you will see that we use this query to get the total of comment